<ui version="4.0" >
 <class>EpgBrowserClass</class>
 <widget class="QWidget" name="EpgBrowserClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Browser</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/icons/images/browser.png</normaloff>:/icons/images/browser.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QToolBar" name="toolBar" >
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <addaction name="actionHome" />
     <addaction name="actionHelp" />
     <addaction name="separator" />
     <addaction name="actionClose" />
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="headerLayout" >
     <item>
      <widget class="QLineEdit" name="addressLine" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonGo" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/go.png</normaloff>:/icons/images/go.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebView" name="epgView" >
     <property name="url" >
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="footerLayout" >
     <item>
      <widget class="QLabel" name="labelStatus" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="maximumSize" >
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionHome" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/home.png</normaloff>:/icons/images/home.png</iconset>
   </property>
   <property name="text" >
    <string>EPG Home</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileclose.png</normaloff>:/icons/images/fileclose.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/help.png</normaloff>:/icons/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>epgView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>399</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>618</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>EpgBrowserClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>epgView</sender>
   <signal>statusBarMessage(QString)</signal>
   <receiver>labelStatus</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>399</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>4</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>epgView</sender>
   <signal>titleChanged(QString)</signal>
   <receiver>EpgBrowserClass</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>399</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
